Makefile.am

```diff
@@ -115,6 +115,7 @@ AM_ETAGSFLAGS = --c++-kinds=+p --fields=+iaS --extra=+q -R --totals=yes --exclud
 AM_CTAGSFLAGS = $(AM_ETAGSFLAGS)
 
 shared_sources = common/FileUtil.cpp \
+                 common/FileUtil-unix.cpp \
```

Comment:
Nit: this commit does multiple splits:

- FileUtil: apple & unix variants
- SigUtil: mobile & server variants
- Util: unix variant

Please, in the future do one split in one commit, so it's easier for the reviewer to see the changes. Thanks.